src/gui/QvisColorTableWindow.C

```diff
@@ -1101,70 +1103,75 @@ QvisColorTableWindow::AddGlobalTag(std::string currtag, bool first_time)
 // 
 //     Justin Privitera, Thu Sep 29 15:22:38 PDT 2022
 //     Replaced braces w/ equals to avoid init list behavior.
+// 
+//    Justin Privitera, Mon Feb 13 14:32:02 PST 2023
+//    Tagging is always enabled, so any code relating to making it optional
+//    has been removed.
 //
 // ****************************************************************************
 
 void
 QvisColorTableWindow::UpdateTags()
 {
-    // We want the 'Default' tag to be checked the very first time tag
-    // filtering is enabled, hence the inclusion of `first_time`.
+    // We want the 'Default' and 'User Defined' tags to be checked the very first time a user
```

Comment:
This sounds like the selected tags are always 'Default' and 'User Defined' when a new VisIt session is started, and there is no way for a user's tag selection to be saved for the next time they run. Is that a correct understanding ?

Reply:
> This sounds like the selected tags are always 'Default' and 'User Defined' when a new VisIt session is started, and there is no way for a user's tag selection to be saved for the next time they run. Is that a correct understanding ?

No, I can verify that this works as intended - loading a config/session file correctly preserves the tag filtering selection.

As to why - it is because the logic that actually checks 'Default' and 'User Defined' the first time is not run if you have loaded a session/config file.

Reply:
Thank you for the explanation.